docs/serving/using-auto-tls.md

```diff
@@ -6,10 +6,35 @@ type: "docs"
 ---
 
 If you install and configure cert-manager, you can configure Knative to
-automatically obtain new TLS certificates and renew existing ones. To learn more
-about using secure connections in Knative, see
+automatically obtain new TLS certificates and renew existing ones for Knative 
+Service.
+To learn more about using secure connections in Knative, see
 [Configuring HTTPS with TLS certificates](./using-a-tls-cert.md).
 
+## Automatic TLS provision mode
+
+In Knative, we support the following modes of Auto TLS:
+
+1.  Using DNS01 challenge
+
+    In this mode, your cluster needs to be able to talk to your DNS server to verify the ownership of your domain.
+    Specifically, when using DNS challenge, we support:
```

Comment:
Replace "Specifically....per namespace" with something like "Provision Certificate per namespace is supported when using DNS challenge mode."

"In this mode, a single Certificate will be provisioned per namespace and **is** reused across the Knative Services within the same namespace."
Remove 'if possible' and instead explain any caveats where this might not be possible.

Remove "If you want to have a fast certificate provision process, this way is recommended." and instead have a similar comment at the beginning of the explanation, e.g.
 "Provision Certificate per namespace is supported when using DNS challenge mode. This is the recommended mode for faster certificate provision."